weather_mv/loader_pipeline/util_test.py

```diff
@@ -42,4 +43,17 @@ def test_no_duplicate_coordinates(self):
 
 class ToJsonSerializableTypeTests(unittest.TestCase):
     # TODO(#106): Write tests...
-    pass
+    def assert_serializes_json(self, input):
+        try:
+            json.loads(json.dumps(input))
+            print("Successfully passed!")
+        except json.JSONDecodeError:
+            self.fail()
+
+    def test_numpy_float_becomes_float(self):
+        res = to_json_serializable_type(np.float(42))
+        self.assert_serializes_json(res)
```

Comment:
This assert is not sufficient. Please test that `res` is of type `float` and equals `42`.